polars/polars-io/src/csv_core/parser.rs

```diff
@@ -557,4 +586,36 @@ mod test {
         assert_eq!(fields2.next(), Some(("12345".as_bytes(), false)));
         assert_eq!(fields2.next(), None);
     }
+
+    #[test]
+    fn trim_must_yield_empty_slice_when_blank() {
+        // when only blanks
+        let input = "     ";
+        let trimmed = trim(input.as_bytes());
+        assert!(trimmed.is_empty());
+
+        // when only tabs
+        let input = "\t\t\t\t";
+        let trimmed = trim(input.as_bytes());
+        assert!(trimmed.is_empty());
+
+        // when both are mixed
+        let input = "\t  \t  \t  \t";
+        let trimmed = trim(input.as_bytes());
+        assert!(trimmed.is_empty())
+    }
+
+    #[test]
+    fn trim_must_not_alter_data_when_no_blank() {
+        let input = "oh boy !";
+        let trimmed = trim(input.as_bytes());
+        assert_eq!(input.as_bytes(), trimmed);
+    }
+    #[test]
+    fn trim_only_drops_leading_and_trailing_blanks() {
+        let input = "\t  \toh boy !\t \t ";
+        let expect = "oh boy !";
+        let trimmed = trim(input.as_bytes());
+        assert_eq!(input.as_bytes(), trimmed);
```

Comment:
```suggestion
        assert_eq!(expected.as_bytes(), trimmed);
```
Shouldn't this be this?

Reply:
You're right, sorry ! 
Actually, the variable i called `expect` so it should be as follows:
```
assert_eq!(expect.as_bytes(), trimmed);
```